### ISSUE TITLE
Copy Linked Record

### ISSUE THREAD

**Issue by MikeMillerGIS** [Esri/local-government-desktop-addins#102]
Reported on Geonet: 
one AA rule we're working on for signs is copying linked record between the sign point and sign__attach table to fill in the sign ID (FacilityID) so we can recognize IDs that we're familiar with. We've tried the method with GLOBALID|REL_GLOBALID but that fails, however; substituting a different ID for the PK/FK relate works. 

https://geonet.esri.com/message/596805?et=watches.email.thread#comment-596805

**Comment by MikeMillerGIS** [comment]
More details:
Awesome! Thanks. I had a fleeting thought that I was to remember to do this. So, here's the set up using the LGIM:
FacilitiesStreets/Sign feature with the FACILITYID column (text 20)
in the Sign_ATTACH table we added a column: FACILITYID (text 20)
We wanted copy linked record to drive the FACILITYID from Sign to Sign_ATTACH using the PK/FK of Sign.GlobalID and Attach.REL_GLOBALID
This doesn't work. But if we changed the method to fill in a different field and PK/FK on FACILITYID that we manually entered, it works. Makes me suspicious that it doesn't like to use the GlobalID.
Also, with the attachments enabled there is the relationship class in the LGIM--don't know if that affects the AA method.
 
DynamicValue example:
Sign__Attach|FACILITYID|COPY_LINKED_RECORD|Sign|FACILITYID|GlobalID|REL_GLOBALID
